helm/values.yaml

```diff
@@ -81,11 +81,22 @@ securityContext: {}
 
 contrib:
   agents:
+    k8s:
+      enabled: true
     istio:
       enabled: true
     helm:
       enabled: true
     observability:
+      prometheus:
+        url: ""
```

Comment:
I wonder if we should default this to `grafana.grafana:<port>` or something like that, so that if we have a doc it "just works"